### PR TITLE
Update Description to remove references to ODH

### PR DESCRIPTION
**Pull request by anishasthana**
SSIA

### CODE REVIEW

**Review by oindrillac — COMMENTED** [review]
/lgtm

File: content/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-description: "Operate First for ODH"
+description: "Open Source Operations for the Cloud"
```
Comment (quaid) [inline content/index.md:2]: Thanks for noticing and fixing this!

I like this description for sure, but I also wonder if we could find a way to include more aspects of the community and our principles in that space?

Some ideas to spark more:

- "Improving open source through open operations"
- "Merging operations and open source development"
- "Incorporating operational experience into software development"

What do you think? Am I trying too hard? 😁 Or does that make some sense?

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Review by 4n4nd — COMMENTED** [review]
/lgtm

File: content/index.md (hunk 1 above)
Comment (quaid) [reply content/index.md:2]: Apparently I never submitted my review that I wrote yesterday.

I'll think on this and maybe make a new PR with a more holistic description.

**Pull request merged**